docs/UserGuide.md

```diff
@@ -359,40 +393,140 @@ A person can have any number of tags and insurance policies (including 0).
+
+**Examples**:
+* Add a meeting and there are no clashes.
+    * `meet 1 20.05.2021 15:00 16:00 MRT`
+
+      ![meet-add](images/meet-add.png)
+
+* Add a meeting but there are clashes.
+    * `meet 3 -add 20.05.2021 15:30 17:30 MRT`
+
+      ![meet-clash](images/meet-clash.png)
+
+* Delete a meeting.
+    * `meet 1 -delete 20.05.2021 15:00 16:00 MRT`
+
+* Clear all meetings.
+    * `meet 2 -clear`
+
+[Return to Table of Contents](#table-of-contents)
+<br><br>
+
+### <span style="color:#f75c66">Contact Viewing Features</span>
+Contact viewing features allow you to view your client contacts in different ways. For example,
+you may reorder your contact list or look for specific clients.
 
 ### `list`: List all clients 
 
-**Purpose**: Shows a list of all clients in ClientBook. Optional identifiers can be specified to show a list with only the desired attributes.
+**Purpose**: Shows a list of all clients in ClientBook. You may use optional identifiers in conjunction with the minus (-)
+symbol to select which client details you want to see, so that you can have a clutter free view of your client contacts.
 
-**Format**: `list [-n] [-p] [-e] [-a] [-i] [-t] [-m]`
+**Format**: `list [-p] [-e] [-a] [-i] [-m]`
 
 **Examples**: 
 *  `list` without any specified identifiers shows a list of all clients and all their information.
-  * `list`
-    
-    ![list](images/list.png)
+
+![list](images/list.png)
     
-*  One or more identifiers can be used to make `list` only show the specified information. The following command shows a list of all clients and their phone number and insurance policy number.
-   * `list -p -i`
+*  One or more identifiers can be used to make `list` only show the specified information. The following command 
+   shows a list of all clients and their phone number and insurance policy number. A client's name and tags will
+   be shown regardless of the identifiers that you specify.
+   
+`list -p -i`
     
-     ![list phone](images/list-phone-policy-annotate.png)
+![list phone](images/list-phone-policy-annotate.png)
 
 [Return to Table of Contents](#table-of-contents)
 <br><br>
 
   
 ### `find`: Search for client contact based on keywords
 
-**Purpose**: Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
-Optional identifiers can be added to show the list of matched clients with only the specified field(s), similar to the [`list`](#list-list-all-clients) command.
+**Purpose**: Finds client contacts based on the attribute and keywords that you specify.
+You may use optional identifiers in conjunction with the minus(-) symbol to limit the details shown, similar to the [`list`](#list-list-all-clients) command.
 
 **Format**: `find IDENTIFIER/KEYWORD [& KEYWORDS]…​ [-IDENTIFIER]…​`
 
+* The `IDENTIFIER` specifies which attribute of the client you want to search for. E.g. if you want to search for a client contact
```

Comment:
Do you think it would be clearer to shift the "E.g. ..." to the next line? Else someone who is looking through it quickly might mistake it to be "for. E.g." to mean "for example". Perhaps can consider using "For example" instead of "E.g.", since this is part of a sentence.

Reply:
done.